scenarios/kubernetes_bazel.py

```diff
@@ -157,10 +164,14 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_build = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_build)
+                # log push-build location to path child jobs can find
+                # (gs://<shared-bucket>/$PULL_REFS/bazel-build-location.txt)
+                pull_refs = os.getenv('PULL_REFS', '')
+                gcs_shared = args.gcs_shared + pull_refs + '/bazel-build-location.txt'
```

Comment:
it'd probably be safer to use `os.path.join` here so that you don't end up with something weird if `--gcs_shared` is missing a trailing slash. i.e. change this to
```python
gcs_shared = os.path.join(args.gcs_shared, pull_refs, 'bazel-build-location.txt`)
```